### PR TITLE
MWPW-113016 - Initial Library Support

### PR DESCRIPTION
**Pull request by auniverseaway**
* Supports block library
* Basic plumbing for tokens, templates, placeholders
* Triggered via Sidekick

Resolves: [MWPW-113016](https://jira.corp.adobe.com/browse/MWPW-113016)

### CODE REVIEW

**Comment by auniverseaway** [comment]
In order to test this, you need to use the sidekick bookmarklet...

```
javascript:/* ** Helix Sidekick Bookmarklet ** */(() %3D> %7Bconst c%3D%7B"owner":"adobecom","host":"localhost:3000","repo":"milo","ref":"library","hlx3":true%7D%3Bconst s%3Ddocument.createElement(%27script%27)%3Bs.id%3D%27hlx-sk-app%27%3Bs.src%3D%27https://www.hlx.live/tools/sidekick/app.js%27%3Bs.dataset.config%3DJSON.stringify(c)%3Bif(document.getElementById(%27hlx-sk-app%27))%7Bdocument.getElementById(%27hlx-sk-app%27).replaceWith(s)%3B%7D else %7Bdocument.head.append(s)%3B%7D%7D)()%3B
```

File: libs/ui/library/library.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+      stub(list);
+      break;
+    case 'Placeholders':
+      stub(list);
+      break;
+    case 'Tokens':
+      stub(list);
+      break;
+    default:
+      console.log('no list');
```
Comment (fullcolorcoder) [inline libs/ui/library/library.js:113]: Is this a good LANA opportunity?

File: libs/ui/library/library.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    const insetEls = finder.querySelectorAll('.inset');
+    insetEls.forEach((el) => {
+      el.classList.remove('inset');
+    });
+    finder.classList.remove('allow-back');
+  });
+  header.append(back);
+  finder.append(header);
+
+  // Container
```
Comment (chrischrischris) [inline libs/ui/library/library.js:137]: same

**Review by chrischrischris — APPROVED** [review]

File: libs/ui/library/library.js (hunk 1 above)
Comment (auniverseaway) [reply libs/ui/library/library.js:113]: I have to ask @chrischrischris some questions on the best way to do this. I'll have another PR to fix it tomorrow.

**Pull request merged**